packages/web-components/fast-foundation/src/avatar/avatar.template.ts

```diff
@@ -10,19 +10,10 @@ export function avatarTemplate<T extends FASTAvatar>(
     options: AvatarOptions = {}
 ): ElementViewTemplate<T> {
     return html<T>`
-    <div
-        class="backplate"
-        part="backplate"
-    >
-        <a
-            class="link"
-            part="link"
-            href="${x => (x.link ? x.link : void 0)}"
-        >
+        <div class="backplate" part="backplate">
```

Comment:
Given the removal of link, can the backplate simply be the host element? Seems like this might be redundant with that change.

Reply:
Since that was more style-breaking I was planning to leave that in the main foundation template update PR.